### PR TITLE
feat: activities were not shown due to lendCta being mounted before k…

### PR DESCRIPTION
**Pull request by christian14b**
…nowing if experiment was enabled. Also, adding remaining props to kvLoanActivities

### CODE REVIEW

**Review by roger-in-kiva — CHANGES_REQUESTED** [review]
addToBasket emit is not working as expected...

File: src/components/BorrowerProfile/LendCta.vue [hunk 1]
```diff
@@ -570,8 +574,9 @@ export default {
 		},
 	},
 	methods: {
-		async addToBasket() {
+		async addToBasket(lendAmount = 0) {
```
Comment (roger-in-kiva) [inline src/components/BorrowerProfile/LendCta.vue:577]: Working fine from the modal but BP lend cta is broken...

File: src/components/BorrowerProfile/LendCta.vue [hunk 2]
```diff
@@ -585,11 +590,11 @@ export default {
 				}
 			}).catch(e => {
 				this.isAdding = false;
-				const msg = e[0]?.extensions?.code === 'reached_anonymous_basket_limit' && e[0]?.message
+				this.errorMsg = e[0]?.extensions?.code === 'reached_anonymous_basket_limit' && e[0]?.message
```
Comment (roger-in-kiva) [inline src/components/BorrowerProfile/LendCta.vue:593]: We need to clean this when addToBasket method is called.

**Review by roger-in-kiva — APPROVED** [review]

File: src/components/BorrowerProfile/LendCta.vue [hunk 3]
```diff
@@ -417,9 +421,9 @@ export default {
 			type: Boolean,
 			default: false,
 		},
-		enableActivityFeed: {
-			type: Boolean,
-			default: false,
+		activities: {
+			type: Object,
+			default: () => ({}),
```
Comment (dyersituations) [inline src/components/BorrowerProfile/LendCta.vue:426]: If we are checking on whether this is defined, then this default should also be `null`

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 4]
```diff
@@ -594,6 +596,12 @@ export default {
 		);
 		if (activityFeedExpData?.version === 'b') {
 			this.enableActivityFeed = true;
```
Comment (dyersituations) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:598]: Can this flag be removed now that we aren't checking it?

**Review by dyersituations — APPROVED** [review]

**Pull request merged**